src/material-experimental/mdc-slider/slider.ts

```diff
@@ -70,6 +70,101 @@ export const MAT_SLIDER_VALUE_ACCESSOR: any = {
   multi: true
 };
 
+class SliderAdapter implements MDCSliderAdapter {
+  constructor(private _delegate: MatSlider) {}
+
+  hasClass = (className: string) =>
```

Comment:
Use class member syntax, otherwise you're still creating per-instance closures and not really gaining anything.

example:

hasClass(className: string) {
  return this._delegate.getElementRef().nativeElement.classList.contains(className);
}

Reply:
Yes, just added that.

Reply:
Looks like you still need to push the change?